alerts/cluster-monitoring-operator/TargetDown.md

```diff
@@ -55,13 +55,40 @@ We are ready to diagnosing the root cause by inspecting each component on the sc
 workflow.
 
 ### Potential Issues and Resolutions
+
+#### Unknown Content Type
+
+Beginning with OCP `4.19`, Prometheus is more strict concerning the `Content-Type`
+header received when scraping, if the target being scraped does not specify
+a `Content-Type` header or if the header is unparsable or unrecognised and
+no [`fallback_scrape_protocol`] is set for the target, the scrape is considered
+as having failed, which would lead to the alert being fired.
+
+For more context, error logs similar to the one below should be present:
+
+   ```shell
+   $ NAMESPACE='<value of namespace label from alert>'
+
+   $ oc -n $NAMESPACE logs -l 'app.kubernetes.io/name=prometheus' --tail=-1 | \
+   grep 'Failed to determine correct type of scrape target.*' \
+   | sort | uniq -c | sort -n
+   msg="Failed to determine correct type of scrape target." \
+   component="scrape manager" \
+   scrape_pool=<monitor>/<namespace>/<service-monitor-name>/<endpoint-id> \
+   target=concerned-target content_type="concerned-content_type" \
+   fallback_media_type="fallback_media_type-if-any" err="explicit_error"
+   ```
+
+In such a case, please open a support case.
```

Comment:
maybe exclude `foo-bar` namespaces with unjustified `openshift.io/cluster-monitoring=true`?
Or maybe that could be done by support?